download.py

```diff
@@ -0,0 +1,53 @@
+import os
+import wget
+import requests
+from github import Github
+
+
+def download_squad_assets():
+    directory = 'data/SQuAD_1.1'
+    squad_urls = [
+        'https://rajpurkar.github.io/SQuAD-explorer/dataset/train-v1.1.json',
+        'https://rajpurkar.github.io/SQuAD-explorer/dataset/dev-v1.1.json',
+    ]
+
+    if not os.path.exists(directory):
+        os.makedirs(directory)
+
+    print("Downloading SQuAD v1.1 data...")
+
+    for squad_url in squad_urls:
+        wget.download(url=squad_url, out=directory)
+
+def download_models():
+    directory = 'models'
+    models_url = [
+    'https://github.com/cdqa-suite/cdQA/releases/download/bert_qa_vGPU/bert_qa_vGPU-sklearn.joblib',
+    'https://github.com/cdqa-suite/cdQA/releases/download/bert_qa_vCPU/bert_qa_vCPU-sklearn.joblib'
+    ]
+
+    print('\nDownloading trained models...')
```

Comment:
`\n`? haha

Reply:
Yes, we need to jump a line, if we don't we will have this: 

![image](https://user-images.githubusercontent.com/43521764/60108834-98a7ba00-9769-11e9-9ae5-12ba0a9ba9f9.png)

Reply:
Oh i see now sorry 😜